src/mavis/summary/summary.py

```diff
@@ -190,45 +192,120 @@ def group_by_distance(calls, distances):
     return grouped_calls, removed_calls
 
 
-def annotate_dgv(bpps, dgv_regions_by_reference_name, distance=0):
+def annotate_with_dgv_bpp(
+    bpps: List[BreakpointPair],
+    dgv_regions_list: List[BreakpointPair],
+    input_cluster_radius,
+):
     """
-    given a list of bpps and a dgv reference, annotate the events that are within the set distance of both breakpoints
+    Given a list of breakpoint pairs (bpps) and bpps from another reference set), annotate the events that are within the set distance of both breakpoints
 
     Args:
         bpps (list) : the list of BreakpointPair objects
-        dgv_regions_by_reference_name (dict) : the dgv reference regions file loaded by load_masking_regions
-        distance (int) : the minimum distance required to match a dgv event with a breakpoint
+        dgv_regions_list (list) : list of BreakpointPair objects specified by the MAVIS input file
+        cluster_radius (int) : Distance used in matching input SVs to reference SVs through clusterind, defined by summary.cluster_radius in the configuration file
```

Comment:
argument names here do not match the function signature. While you're fixing those, move any types to the function signature instead of the comments

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).